specification.md

```diff
@@ -1631,6 +1631,25 @@ There are two types of conditions for switch states:
 
 These are exclusive, meaning that a switch state can define one or the other condition type, but not both.
 
+At times multiple defined conditions can be evaluated to `true` by runtime implementations.
+This can lead to ambiguity as to which condition should have its transition executed next.
+The specification has two ways of dealing with this ambiguity on the markup level:
+
+  1. order-based: conditions defined first take precedence over conditions defined later.
+  This is backed by the fact that arrays/sequences are orderd in both JSON and YAML.
+  2. priority-based: conditions with higher priority value take precedence over conditions with lower priority.
+  This value is taken from a condition's `priority` parameter.
+
+For example, let's say there are two conditions, A and B, defined in that order. If both A and B are satisfied
+then the A's transition must be executed. On the other hand, if A had a priority of 10 and B's priority was 20
+then B's transition would be executed instead.
+
+If the `priority` parameter of one of the defined conditions is specified, all conditions in the same Switch State
+must also specify it. If the user fails to do so, the implementation must notify them of this.
+
+If two conditions in the same Switch State define the same value for the `priority` parameter, this is considered
+a logical error and implementations must notify users of this.
```

Comment:
Change "implementations" to "runtime implementations" to be clear please. Same above (there are two places)